CONTRIBUTING.md

```diff
@@ -0,0 +1,292 @@
+Contributing to InfluxDB v2
+========================
+
+Bug reports
+---------------
+Before you file an issue, please search existing issues in case it has already been filed, or perhaps even fixed. If you file an issue, please include the following.
+* Full details of your operating system (or distribution) e.g. 64-bit Ubuntu 14.04.
+* The version of InfluxDB you are running
+* Whether you installed it using a pre-built package, or built it from source.
+* A small test case, if applicable, that demonstrates the issues.
+
+Remember the golden rule of bug reports: **The easier you make it for us to reproduce the problem, the faster it will get fixed.**
+If you have never written a bug report before, or if you want to brush up on your bug reporting skills, we recommend reading [Simon Tatham's essay "How to Report Bugs Effectively."](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
+
+TEST CASES SECTION NEEDS REVISION based on v2 API
+
+Test cases should be in the form of `curl` commands. For example:
+```bash
+# create database
+curl -X POST http://localhost:8086/query --data-urlencode "q=CREATE DATABASE mydb"
```

Comment:
Yep, these all need to be adjusted

Reply:
@e-dard @8none1 what exactly need to be adjusted ?

Reply:
@Elbehery everything. That doesn't look at all like the API we have in 2.x. There should be some Swagger documentation somewhere that can help you find the correct API endpoints. The port is now `9999`.

Reply:
@e-dard  ok but the test case should always use `curl`, only the endpoint changes, correct ?